script.js

```diff
@@ -0,0 +1,72 @@
+"use strict"
+const USD = "USD"
+const EUR = "EUR"
+let requestRate = +prompt(`введите актуальный курс`)
+if (!isNaN(requestRate) && requestRate > 0) {
+     function createCurrencyConverter(requestRate) {
+        let currencyRate = requestRate
+
+        function intoEUR(valueUSD) {
+            let resultEUR = valueUSD * currencyRate
+            return resultEUR.toLocaleString( 'es-ES', {style: 'currency' , currency: 'EUR'})
+        }
+
+        function intoUSD(valueEUR) {
+            let resultUSD = valueEUR / currencyRate
+            return resultUSD.toLocaleString( 'en-US', {style: 'currency' , currency: 'USD'})
+        }
+
+        return {
+            intoEUR: intoEUR,
+            intoUSD: intoUSD,
+        }

```

Comment:
можно сократить до ( но мы это будем еще в будущем проходить )

```suggestion
        return {
            intoEUR,
            intoUSD,
        }

```